index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <nav id="navbar">
+    <ul>
+      <li><a href="#projects">Projects</a></li>
+      <li><a href="#">About</a></li>
+      <li><a href="#contact">Contact</a></li>
+
+    </ul>
+  </nav>
```

Comment:
## Optional

- [ ] Please you need to phrase the comments politely, as "suggestions" instead of orders.
- This is a polite suggestion: I think it would be better to use a `header`...

- [ ]  Always includes an explanation, a "why", for this requested change